R/draws-index.R

```diff
@@ -221,28 +231,10 @@ nvariables.NULL <- function(x) {
 }
 
 #' @export
-nvariables.draws_matrix <- function(x) {
-  NCOL(x)
-}
-
-#' @export
-nvariables.draws_array <- function(x) {
-  dim(x)[3]
-}
-
-#' @export
-nvariables.draws_df <- function(x) {
+nvariables.draws <- function(x) {
```

Comment:
Should this also take a `reserved` argument and then pass it to `variables()`?

Reply:
Done.